qubes/tests/integ/dispvm.py

```diff
@@ -879,6 +879,41 @@ def _handle_editor(self, winid, copy=False):
         else:
             raise KeyError(window_title)
 
+    def _whonix_ws_dispvm_confirm(self, action_str):
+        try:
+            winid = self.wait_for_window(
+                "Operation execution",
+                search_class=True,
```

Comment:
search_class=True means to look based on WM_CLASS property instead of the title. You can do either here, but with search_class=True, you need to actually use window class (xprop says it's `qrexec-policy-agent`)